ttnn/cpp/ttnn/operations/eltwise/unary/device/unary_device_operation.cpp

```diff
@@ -30,8 +30,8 @@ void validate_supported_arch_dtype(
         case UnaryOpType::FILL:
            if(arch == tt::ARCH::GRAYSKULL){
                 TT_FATAL(
-                    input_datatype == DataType::BFLOAT16,
-                    "Supported input data type '{}' for UnaryOpType '{}' on Grayskull.",
+                    (input_datatype == DataType::BFLOAT16 || input_datatype == DataType::BFLOAT8_B),
+                    "Unsupported input data type '{}' for UnaryOpType '{}' on Grayskull.",
```

Comment:
Let's tell a user what types are supported

Reply:
also, does it mean we support any type on wh+?

Reply:
for WH, we support BFLOAT16, BFLOAT8_B, FLOAT_32. for other dtypes, we'll rely on current implementation for now